config/core/deployments/webhook.yaml

```diff
@@ -51,11 +51,9 @@ spec:
 
         resources:
           requests:
-            # taken from serving.
-            cpu: 100m
-            memory: 50Mi
+            cpu: 20m
+            memory: 25Mi
           limits:
```

Comment:
not sure I think it's a good idea to decrease them, just b/c of `kind`... 

/hold

Generally adding numbers here by random guessing is not right.  I think they were added b/c of the `kube linter` by @julz 

Perhaps when we add numbers, we should be sure it works fine on prod envs too

Have in mind folks are already using those, not sure if havng a small nr like that is great idea

Reply:
Just to be clear, many of the requests/limits modified in this PR are actually modifying long-existing values (see [version in latest release](https://github.com/knative/eventing/blob/release-0.24/config/core/deployments/webhook.yaml#L53-L60)).  Most things already (correctly) had a request/limit, and most had based it on the same values serving uses for its controllers. The PRs just added them to a couple things that didn't (mt-channel-broker, in-memory-channel, I think). It's important for things to have a valid request/limit because otherwise we're relying on luck for things not to land on over-full nodes and give people a bad time in production.

tldr agree with @matzew; I'd suggest we lower only those _new_ requests introduced in recent PRs (unless we're sure the existing ones were also too high)

Reply:
actually, checking git blame, looks like we [quite intentionally bumped this particular request quite recently](https://github.com/knative/eventing/pull/5112), so it would be a clear regression to drop it back down :(.

Reply:
FWIW, on our on operator, shippng serving/eventing we are already using the extension to set some values to higher values, due to demand, e.g.:
* https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/serving/extension.go#L108-L109
* https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/eventing/extension.go#L55-L56

Reply:
Question: can't we just give `kind` a bit of more horse power? Rather than shrinking the values as suggested?

Reply:
These are requests, not limits, so they just are a baseline, so I don't think they would cause any problems in prod environments. The limits might, but I didn't change those.

Reply:
but if the requests are too small there's no guarantee the host the instance lands on will have enough spare cpu/mem to actually run the workload. If you tell k8s the app only needs 16MB and it actually needs 64MB that will work until the scheduler happens to put it on a node that only has 16MB free..

Reply:
We don't control the environments that kind runs on, they're provided by GitHub as we're using their runners, they seem to vary slightly and so this is causing the tests to fail